samples/CommunityToolkit.Maui.Markup.Sample/Pages/NewsDetailPage.cs

```diff
@@ -23,17 +23,17 @@ public NewsDetailPage(NewsDetailViewModel newsDetailViewModel) : base(newsDetail
 					.Text("Launch in Browser \uf35d")
 					.Font(size: 20, family: "FontAwesome")
 					.Basis(50)
-					.DynamicResource(Button.TextColorProperty, nameof(BaseTheme.PrimaryTextColor))
-					.DynamicResource(Button.BackgroundColorProperty, nameof(BaseTheme.NavigationBarBackgroundColor))
+					.AppThemeBinding(Button.TextColorProperty, Colors.Black,Color.FromArgb("FFF2EA"))
+					.AppThemeBinding(Button.BackgroundColorProperty, Color.FromArgb("FF6601"), Color.FromArgb("C3560E"))
 					.Bind(Button.CommandProperty, nameof(NewsDetailViewModel.OpenBrowserCommand), BindingMode.OneWay)
 					.SemanticHint("Launches the news article in the devices browser."),
 
 				new Label()
 					.TextCenter()
 					.AlignSelf(FlexAlignSelf.Stretch)
 					.Paddings(bottom: 20)
-					.DynamicResource(Label.TextColorProperty, nameof(BaseTheme.PrimaryTextColor))
-					.DynamicResource(Label.BackgroundColorProperty, nameof(BaseTheme.NavigationBarBackgroundColor))
+					.AppThemeBinding(Label.TextColorProperty, Colors.Black,Color.FromArgb("FFF2EA"))
+					.AppThemeBinding(Label.BackgroundColorProperty, Color.FromArgb("FF6601"), Color.FromArgb("C3560E"))
```

Comment:
Let's keep a constant for `NavigationBarBackgroundColor` since it is used multiple times